docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -308,7 +350,206 @@ NOTE: Starting from **v1.4.0**, if you want to update the Events settings for th
+cloudwatch-logs:
+    - arn: "arn:aws:logs:%AWS_REGION%:%AWS_ACCOUNT_ID%:log-group:%LOG_GROUP_NAME%:*"
+    - arn: "arn:aws:logs:%AWS_REGION%:%AWS_ACCOUNT_ID%:log-group:%LOG_GROUP_NAME%:log-stream:%LOG_STREAM_NAME%"
+ssm-secrets:
+  - "arn:aws:secretsmanager:%AWS_REGION%:%AWS_ACCOUNT_ID%:secret:%SECRET_NAME%"
+kms-keys:
+    - "arn:aws:kms:%AWS_REGION%:%AWS_ACCOUNT_ID%:key/%KMS_KEY_UUID%"
+s3-buckets:
+    - "arn:aws:s3:::%BUCKET_NAME%"
+subnets:
+    - "%SUBNET_ID%"
+security-groups:
+    - "%SECURITY_ID%"
+s3-config-file: "s3://%S3_CONFIG_BUCKET_NAME%/%S3_CONFIG_OBJECT_KEY%"
+continuing-queue:
+    batch_size: 10
+    batching_window_in_second: 0
+
+----
+
+[[direct-publish-config-file-fields]]
+=== Fields
+
+|===
+
+| `kinesis-data-stream.[]` | List of <<aws-serverless-forwarder-inputs-kinesis>> (i.e. triggers) for the forwarder, matching those defined in your <<sample-s3-config-file>>.
+
+| `kinesis-data-stream.[].arn` | ARN of the {aws} Kinesis Data Stream.
+
+| `kinesis-data-stream.[].batch_size` | Set this value above the default (`10`) if you experience ingestion delays in your output *and* `GetRecords.IteratorAgeMilliseconds` and `IncomingRecords` Kinesis CloudWatch metrics for the <<aws-serverless-forwarder-inputs-kinesis>> keep increasing *and* the average execution time of the forwarder is below 14 minutes. This will increase the number of records the forwarder will process in a single execution for the <<aws-serverless-forwarder-inputs-kinesis>>.
+
+| `kinesis-data-stream.[].batching_window_in_second` | Set this value above the default (`0`) if you experience ingestion delays in your output *and* `GetRecords.IteratorAgeMilliseconds` and `IncomingRecords` Kinesis CloudWatch metrics for the <<aws-serverless-forwarder-inputs-kinesis>> keep increasing *and* the average execution time of the forwarder is below 14 minutes. This will increase the number of records the forwarder will process in a single execution for the <<aws-serverless-forwarder-inputs-kinesis>>.
+
+| `kinesis-data-stream.[].starting_position` | Change this value from the default (`TRIM_HORIZON`) if you want to change the starting position of the records processed by the forwarder for the <<aws-serverless-forwarder-inputs-kinesis>>.
+
+| `kinesis-data-stream.[].starting_position_timestamp` | Set this value to the time from which to start reading (in Unix time seconds) if you set `ElasticServerlessForwarderKinesisStartingPosition` to "AT_TIMESTAMP".
+
+| `kinesis-data-stream.[].parallelization_factor` | Set this value above the default (`1`) if you experience ingestion delays in your output *and* `GetRecords.IteratorAgeMilliseconds` and `IncomingRecords` Kinesis CloudWatch metrics for the <<aws-serverless-forwarder-inputs-kinesis>> keep increasing *and* the average execution time of the forwarder is below 14 minutes. This will increase the number of records the forwarder will process in a single execution for the <<aws-serverless-forwarder-inputs-kinesis>>.
```

Comment:
I think we need to remove the term "single execution" here. This parameter is used to increase the number of functions AWS Lambda is allowed to launch for each Kinesis shard.

Reply:
Update from slack:

> | `kinesis-data-stream.[].parallelization_factor` | Defines the number of forwarder functions that can run concurrently per shard (default is `1`). Increase this value if you experience ingestion delays in your output *and* `GetRecords.IteratorAgeMilliseconds` and `IncomingRecords` Kinesis CloudWatch metrics for the <<aws-serverless-forwarder-inputs-kinesis>> keep increasing *and* the average execution time of the forwarder is below 14 minutes. This will increase the number of records processed concurrently for <<aws-serverless-forwarder-inputs-kinesis>>. For more info, refer to https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html[AWS Kinesis docs].